step.yml

```diff
@@ -0,0 +1,100 @@
+#
+# A couple of useful guides & docs:
+#
+# - Main Bitrise CLI docs: https://github.com/bitrise-io/bitrise/tree/master/_docs
+# - Step Development Guideline: https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md
+# - Bitrise.yml format spec: https://github.com/bitrise-io/bitrise/blob/master/_docs/bitrise-yml-format-spec.md
+# - Bitrise docs: http://devcenter.bitrise.io/
+# - Bitrise CLI guides: http://devcenter.bitrise.io/bitrise-cli/
+
+title: |-
+  Custom Test Results Export
+summary: |
+  This step is used to search for test results and to export them for the 'Deploy to Bitrise.io' step. The deploy step can then be used to pick up results and import them into the testing add-on.
+description: |
+  This step is used to search for test results and to export them for the 'Deploy to Bitrise.io' step. The deploy step can then be used to pick up results and import them into the testing add-on.
+website: https://github.com/bitrise-steplib/bitrise-step-custom-test-results-export
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-custom-test-results-export
+support_url: https://github.com/bitrise-steplib/bitrise-step-custom-test-results-export/issues
+host_os_tags:
+  - osx-10.10
+  - ubuntu-16.04
+
+# If this step should be available only for certain project types
+# just uncomment this `project_type_tags` section and include all the
+# project types supported by the step. If the step can be used for all
+# project types then you can just remove this section.
+# If no `project_type_tags` specified (or specified as an empty array)
+# that means the step can be used for any project type.
+# You can find more information about project type tags in the Step Development Guideline:
+# https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md
+#
+# project_type_tags:
+#   - ios
+#   - macos
+#   - android
+#   - xamarin
+#   - react-native
+#   - cordova
+#   - ionic
+
+# Type tags are used for categorizing steps, for easier step discovery in Step Libraries.
+# You can find more information about type tags in the Step Development Guideline:
+# https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md
+type_tags:
+  - test
+
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/bitrise-step-custom-test-results-export
+
+
+inputs:
+  - test_name: Custom test result
```

Comment:
pls do not set a default for this input (users will not realize that they need to set it)